setup.py

```diff
@@ -69,6 +69,7 @@
         'six>=1.9, <2.0',
         'toml>=0.9, <1.0',
         'sseclient==0.0.14',
+        'retrying==1.3.3',
```

Comment:
I meant the opposite. Currently this project defines two packages : [dcos](https://pypi.python.org/pypi/dcos) (under `./dcos`) and [dcoscli](https://pypi.python.org/pypi/dcoscli) (under `./cli`), however that dependency is introduced for the `dcoscli` package, so the change in `cli/requirements.txt` was correct and this one should be reverted.

The reason it also works from here is because `dcoscli` requires `dcos` which would bring that dependency along.